README.md

```diff
@@ -73,39 +39,48 @@ You can try Cadl on the web without installing anything.
 
    It is recommended to have npm 7+. To update npm run `npm install -g npm`
 
-2. Create a folder for your new Cadl project
+2. Install Cadl compiler and libraries:
 
-3. **Via init command:** Run `npx -p @cadl-lang/compiler cadl init` > Select openapi3 library template.
-
-4. **Alternatively manually:** In a command prompt, run the following commands:
-
-   ```
-   cd path\to\cadl\project
+```bash
    npm init -y
    npm install -g @cadl-lang/compiler
-   npm install @cadl-lang/rest @cadl-lang/openapi3
-   ```
+```
 
-   This will create a `package.json` file for your Cadl project and add the necessary Cadl dependencies to it.
+If you do not wish to install the compiler globally with `-g` flag, you will need to install it locally once in every Cadl project folder. You would also need to prefix every Cadl run command with `npx`. See [npx documentation](https://docs.npmjs.com/cli/v7/commands/npx)
 
-5. Install the Cadl extension for your editor of choice:
+```bash
+    npx cadl init
+    npx cadl compile
+```
+
+3. Install the Cadl extension for your editor of choice:
 
    - [Instructions for Visual Studio](#installing-visual-studio-extension)
    - [Instructions for Visual Studio Code](#installing-vs-code-extension)
 
-6. Open the folder in your editor and create a new file `main.cadl`
+### Creating Cadl project
+
+1. Create a folder for your new Cadl project and install libraries in local folder:
```

Comment:
Is there some reason to do this npm install before the cadl init? If you do the cadl init first, then you just need to do npm install (with no libraries) afterwards and it will pick up the dependencies that cadl init put in package.json.